testsuite/libffi.closures/cls_24byte.c

```diff
@@ -118,7 +118,7 @@ int main (void)
   args_dbl[4] = NULL;
 
   ffi_call(&cif, FFI_FN(cls_struct_24byte_fn), &res_dbl, args_dbl);
-  /* { dg-output "9 2 6 5 1 2 3 7 4 5 7 9 8 6 1 4: 22 15 17 25" } */
+  /* { dg-output "9 2 6 5 1 2 3 7 4 5 7 9 8 6 1 9: 22 15 17 25" } */
```

Comment:
I'm pretty sure that should correctly be a 4:
```
  struct cls_struct_24byte e_dbl = { 9.0, 2.0, 6, 5.0 };
  struct cls_struct_24byte f_dbl = { 1.0, 2.0, 3, 7.0 };
  struct cls_struct_24byte g_dbl = { 4.0, 5.0, 7, 9.0 };
  struct cls_struct_24byte h_dbl = { 8.0, 6.0, 1, 4.0 };
```
So:
9 2 6 5 | 1 2 3 7 | 4 5 7 9 | 8 6 1 **9**
should be
9 2 6 5 | 1 2 3 7 | 4 5 7 9 | 8 6 1 **4**
kinda surprises me that nobody noticed this yet upstream...

Reply:
Oh huh, curious. Perhaps upstream's CI failures were related to this?

Sorry about the this change, I had reverted the entire test suite to see which `dg-output`/`printf` lines had changed.

Reply:
See https://github.com/libffi/libffi/pull/652